resources/service_systemd.rb

```diff
@@ -99,4 +100,17 @@ def create_init
       action :nothing
     end
   end
+
+  def delete_runit
```

Comment:
It would be nice to wrap this whole thing in a file.exists so we can avoid showing this the list of resources if the runit service was never setup